pages/checkout/index.html

```diff
@@ -1,15 +1,15 @@
 ---
 layout: default
-title: Become a Gruntwork Subscriber
-excerpt: Configure the plan just right for your team.
+title: Customize your plan with add-ons
+excerpt: We offer professional support and a reference architecture.
```

Comment:
Oh, clever... So `/pricing` is now the first step of the check out flow, and `/checkout` is step 2? And how do you get to step 3? By clicking the red "Checkout" button?

Reply:
Yes.